configure_metrics_agent.sh

```diff
@@ -1,6 +1,120 @@
-#!/bin/bash
+#!/bin/bash -e
 
 echo "Configuring agent..."
 BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+INSTALL_DIR=`cd $BIN_DIR/.. && pwd`
 
-${BIN_DIR}/python ${BIN_DIR}/monasca-setup $@
\ No newline at end of file
+sudo mkdir -p /etc/monasca
+
+sudo ${BIN_DIR}/python ${BIN_DIR}/monasca-setup $@
+
+generate_supervisor_config() {
+    local tmp_conf_file="/tmp/supervisor.conf"
+    local agent_dir="/etc/monasca/agent"
+    local supervisor_file="$agent_dir/supervisor.conf"
+
+    echo "[Unit]
```

Comment:
supervisor.conf is a mess :)
It includes some parts of a service file and also some bash output. Also paths for monasca-collector, monasca-forwarder and monasca-statsd are incorrect

Reply:
Fixed. PTAL.